src/SCTPWrapper.cpp

```diff
@@ -98,6 +98,42 @@ void SCTPWrapper::OnNotification(union sctp_notification *notify, size_t len) {
       break;
     case SCTP_STREAM_RESET_EVENT:
       SPDLOG_TRACE(logger, "OnNotification(type=SCTP_STREAM_RESET_EVENT)");
+      struct sctp_stream_reset_event reset_event;
+      reset_event = notify->sn_strreset_event;
+      for (int i = 1; i < 2; i++) {
```

Comment:
Fix in https://github.com/chadnickbok/librtcdcpp/pull/24/commits/289b7eff6b8461ba8591e3159532cee15350e686